templates/empty_ts/build.ts

```diff
@@ -3,17 +3,20 @@ gulp.task("build", async () => require(["asc"], asc => {
   const args = [
     "main.ts",
     "--baseDir", "src",
+    "--textFile", "../out/main.wast",
     "--binaryFile", "../out/main.wasm",
+    "--asmjsFile", "../out/main.asm.js",
     "--sourceMap",
     "--measure"
   ];
 
+  project.setStatus("Building Project ...");
```

Comment:
I don't think it's necessary to set the status here, a call to `setStatus` appears in the `App.tsx`  `build` function.

Reply:
Yeah, already removed it :)